test/plugins/features/movies/controller.test.js

```diff
@@ -12,16 +12,145 @@ describe('movie controller', () => {
 
       return Controller.create(payload)
       .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
+        expect(movie.get('name')).to.eql(payload.title);
 
         return new Movie({ id: movie.id}).fetch();
 
       })
       .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
+        expect(movie.get('name')).to.eql(payload.title);
       });
     });
 
   });
-  
+
+  describe('retrieve a list of movies', () => {
+
+    it('returns all movies', () => {
+      let count;
+      return new Movie().fetchAll()
+      .then((movies) => {
+        return movies.length;
+      })
+      .then((c) => {
+        count = c;
+        return Controller.findAll();
+      })
+      .then((movies) => {
+        console.log(movies);
+        expect(movies).to.exist;
+        expect(movies).to.have.length(count);
+      });
+    });
+
+  });
+
+  describe('filters movies based on specified dates', () => {
+
+    it('returns movies in 2000', () => {
+      let count;
+      return new Movie().query(function(qb) {
+        qb.where('release_year', 2000);
+      })
+      .fetchAll()
+      .then((movies) => {
+        return movies.length;
+      })
+      .then((c) => {
+        count = c;
+        return Controller.findAll({release_year: 2000});
```

Comment:
Nit: Generally we pad the brackets in object literals with spaces. So this would become:
`{ release_year: 2000 }`. This goes for the other object literals in the file.